glib/src/param_spec.rs

```diff
@@ -384,6 +384,98 @@ macro_rules! define_param_spec_numeric {
     };
 }
 
+macro_rules! define_builder {
+    (@constructors $rust_type:ident, $builder_type:ident $(($($req_ident:ident: $req_ty:ty,)*))?) => {
+        impl<'a> $builder_type<'a> {
+            pub fn new(name: &'a str, $($($req_ident: $req_ty)*)?) -> Self {
+                Self {
```

Comment:
I would make the `new()` function of the builder private and only keep the `builder()` function on the pspec. Having both is mildly confusing IMHO